stdlib/config-api/src/test/java/org/ballerinalang/stdlib/config/ComplexConfigKeyTest.java

```diff
@@ -42,8 +41,7 @@
 
     @BeforeClass
     public void setup() throws IOException {
-        resourceRoot = new File(getClass().getProtectionDomain().getCodeSource().getLocation().getPath())
-                .getAbsolutePath();
+        resourceRoot = Paths.get("src", "test", "resources").toAbsolutePath().toString();
```

Comment:
Why  was this changed? The intention of the previous one was to take the sources and the config files from the `/target` directory. This will take it straight from the `src/test/resources` directory. This may be problematic in cases where the files get deleted.

Reply:
How built artifacts structured in output directory(/target, /build) is different in maven and gradle. We can't use the same way to get test resources in gradle. So we need to get it from src/test/resources directory. 
Do we have any use case to delete test resources?

Reply:
Yes, there's a test for reading the secret for decrypting through a txt file. This secret file is deleted after reading once. If we point it to the resources dir, the resource will be deleted.